frontend_tests/casper_tests/06-settings.js

```diff
@@ -99,6 +90,15 @@ casper.then(function () {
     });
 });
 
+casper.then(function () {
+    casper.waitUntilVisible('#account-settings-status', function () {
+        casper.test.assertSelectorHasText('#account-settings-status', 'Updated settings!');
+
+        casper.click('[data-section="your-bots"]');
```

Comment:
okay, so here the lines 56-91 are the actual ones responsible for testing `download personal API key`. The lines which have been moved here are responsible for taking the control to "your-bots" section from the "your-accounts" section.